libs/core-styles/src/lib/_imports/objects/o-site.css

```diff
@@ -1,26 +0,0 @@
-/*
-Site
-
-Layout and class names that are applicable to most websites:
-
-- ["sticky footer"](https://css-tricks.com/couple-takes-sticky-footer/)
-
-Markup: o-site.html
-
-Styleguide Objects.Site
-*/
-
-/* SEE: https://css-tricks.com/couple-takes-sticky-footer/#article-header-id-3 */
-.o-site {
-  display: flex;
-  flex-direction: column;
-}
-.o-site__body {
-  flex-grow: 1;
-  flex-shrink: 0;
-  flex-basis: auto;
-}
-.o-site__head,
-.o-site__foot {
-  flex-shrink: 0;
-}
```

Comment:
**Woah, Cowboy!**
Don't delete this, Wes, because there are potential clients:
* new TUP docs can use it if shared header uses iframe solution (see Design Safe usage)
* DesignSafe uses it to give docs iframe a dynamic "full-page" height ([css](https://github.com/DesignSafe-CI/portal/blob/main/designsafe/static/styles/main.css#L935), [markup](https://github.com/DesignSafe-CI/portal/blob/main/designsafe/templates/base.j2#L67))